### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by Brian3647** [Brian3647/snowboard#11]
Overally improve this project's `README.md`, maybe adding screenshots / a logo. Any help is appreciated.

**Comment by ibnz36** [comment]
I don't think this needs a dedicated issue. Will update some aspects of it over the time
